tests/main/uc20-create-partitions-reinstall/task.yaml

```diff
@@ -1,8 +1,14 @@
-summary: Integration tests for the snap-bootstrap binary autodetect
+summary: Integration tests for the bootstrap.Run autodetect
 
 # use the same system and tooling as uc20
 systems: [ubuntu-20.04-64]
```

Comment:
Key sealing is always performed when encrypting and it requires secure boot and tpm to be enabled, so we can use `ubuntu-20.04-64-secboot` here (after snapcore/spread#104 lands and replacing/reconciling with #8539).